frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/EnvironmentVariantsModal/VariantForm/VariantForm.tsx

```diff
@@ -293,7 +293,7 @@ export const VariantForm = ({
                 JSON.parse(payload.value);
             }
             if (variantTypeNumber && payload.type === 'number') {
-                Number(payload.value);
+                return !isNaN(Number(payload.value));
```

Comment:
it shouldn't matter here but in general it's safer to use Number.isNaN over global isNaN